generators/languages/templates/src/main/webapp/i18n/en/_global.json

```diff
@@ -69,16 +69,22 @@
                 <%_ if (clientFramework === 'angular1') { _%>
                 "authenticated": "If you want to <a class=\"alert-link\" href=\"\" ng-click=\"vm.login()\">sign in</a>, you can try the default accounts:<br/>- Administrator (login=\"admin\" and password=\"admin\") <br/>- User (login=\"user\" and password=\"user\").",
                 "register": "You don't have an account yet? <a class=\"alert-link\" href=\"\" ng-click=\"vm.register()\">Register a new account</a>"
-                <%_ } else { _%>
+                <%_ } else if (authenticationType !== 'oauth') { _%>
                 "authenticated": {
-                    "prefix": "If you want to ",
-                    "link": "sign in",
-                    "suffix": ", you can try the default accounts:<br/>- Administrator (login=\"admin\" and password=\"admin\") <br/>- User (login=\"user\" and password=\"user\")."
+                    "prefix": "Please ",
+                    "link": "sign in or register",
+                    "suffix": ".<br><br><strong>NOTE:</strong> You'll have to configure accounts, add them to ROLE_ADMIN/ROLE_USER groups, include groups as a claim, and enable registration on your Identity Provider."
                 },
                 "register": {
                     "noaccount": "You don't have an account yet?",
                     "link": "Register a new account"
                 }
+                <%_ } else { _%>
+                "authenticated": {
+                    "prefix": "If you want to ",
+                    "link": "sign in",
+                    "suffix": ", you can try the default accounts:<br/>- Administrator (login=\"admin\" and password=\"admin\") <br/>- User (login=\"user\" and password=\"user\")."
+                },
```

Comment:
I'm not sure this change is a good idea because we'll have to change all the language files. However, I think it conveys the meaning better since JHipster won't know if registration is enabled on the IdP.